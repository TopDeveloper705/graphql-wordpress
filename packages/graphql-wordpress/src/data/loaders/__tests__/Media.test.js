import getMediaLoaders from 'data/loaders/Media';

const Media = getMediaLoaders();

describe('Test Media loader', () => {
  test('Load a media item', async () => {
    const media = await Media.load(13);
    expect(media.getID()).toMatchSnapshot();
    expect(media).toMatchSnapshot();
  });
});
