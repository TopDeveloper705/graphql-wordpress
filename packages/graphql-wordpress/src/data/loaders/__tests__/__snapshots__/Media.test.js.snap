// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Media loader Load a media item 1`] = `"TWVkaWE6MTM="`;

exports[`Test Media loader Load a media item 2`] = `
Media {
  "id": 13,
}
`;
